ui/app/AppLayouts/Chat/panels/UserListPanel.qml

```diff
@@ -53,61 +50,74 @@ Item {
             right: parent.right
             rightMargin: Style.current.halfPadding
             bottom: parent.bottom
-            bottomMargin: Style.current.bigPadding
         }
 
-        model: SortFilterProxyModel {
-            sourceModel: usersModule.model
-            sorters: [
-                RoleSorter {
-                    roleName: "onlineStatus"
-                    sortOrder: Qt.DescendingOrder
-                },
-                StringSorter {
-                    roleName: "displayName"
-                    caseSensitivity: Qt.CaseInsensitive
-                }
-            ]
-        }
-        section.property: "onlineStatus"
-        section.delegate: (root.width > 58) ? sectionDelegateComponent : null
-        delegate: StatusMemberListItem {
-            width: ListView.view.width
-            nickName: model.localNickname
-            userName: model.displayName !== "" ? model.displayName : model.alias
-            pubKey: Utils.getCompressedPk(model.pubKey)
-            isContact: model.isContact
-            isVerified: model.isVerified
-            isUntrustworthy: model.isUntrustworthy
-            isAdmin: model.isAdmin
-            asset.name: {
-                const isCurrentUser = model.pubKey === root.rootStore.getPubkey()
-                const visibility = Global.privacyModuleInst.profilePicturesVisibility
-
-                if (isCurrentUser
-                        || visibility === Constants.profilePicturesVisibility.everyone
-                        || (visibility === Constants.profilePicturesVisibility.contactsOnly && model.isContact))
-                    return model.icon
-
-                return ""
+        clip: true
```

Comment:
Is this clipping intentional? (with the clip off in the ListView below)?

Reply:
yes, it's intentional, otherwise setting `displayMarginEnd` would have no visual effect.